scvi/model/_autozi.py

```diff
@@ -235,19 +233,18 @@ def get_marginal_ll(
                 )
 
                 # Log-probabilities
-                log_prob_sum = torch.zeros(qz_m.shape[0]).to(self.device)
```

Comment:
has this been tested on a gpu? I remember making this change bc on a GPU the components would live one different devices

Reply:
let me double check, but since the outputs of `forward` should be on the GPU, then all quantities should be on the GPU

Reply:
Hi again @jjhong922 , I confirm that this code runs fine on a GPU (through `test_autozi`)